tutorials/get-started-containers-docker/index.mdx

```diff
@@ -0,0 +1,303 @@
+  ```
+
+  The container ID should match that which you saw with the `docker ps` command.  
+
+8. Run the following command in your terminal to shut down the container. Replace `<container-ID>` with the ID for your container.
+
+  ```
+  sudo docker kill <container-ID>
+  ```
+
+9. Run `sudo docker ps` and/or refresh the endpoint in your browser, to show that the container is no longer running.
+
+## Pushing to the Scaleway Container Registry
+
+To finish, we push the Docker images we have created to a container registry. Container registries are designed to store container images, and make them accessible to those who need them. There are many different container registry platforms, but here we use the [Scaleway Container Registry](https://www.scaleway.com/en/container-registry/).
+
+1. Open a browser and go to the [Scaleway console's Container Registry page](https://console.scaleway.com/registry/namespaces).
+
+2. Follow [these instructions](https://www.scaleway.com/en/docs/compute/container-registry/how-to/create-namespace/) to create a namespace. Make sure that you set your privacy policy to **private**.
+
+  The following steps should be carried out in the terminal of your local machine:
+
+3. Log in to your Container Registry namespace:
+
+  ```bash
+  sudo docker login <address-of-your-namespace> -u nologin -p $SCW_SECRET_TOKEN
+  ```
+
+  <Message type="tip">
+
+  Ensure that you replace `<address-of-your-namespace>` with the address of your own Container Registry namespace (eg `rg.fr-par.scw.cloud/mynamespace`) and that you have [created your API key](https://www.scaleway.com/en/docs/console/my-project/how-to/generate-api-key/) and that you have saved the secret part of your API key as an environment variable e.g. with the command `export SCW_SECRET_TOKEN=a4db08b7-5723-4ba9-8c18-f2de493465a2`
+
+  </Message>
+
+4. Tag the image you want to push with the address of your Container Registry namespace:
+
+  ```bash
+  sudo docker tag mytestapp <address-of-your-namespace>/mytestapp
+  ```
+
+5. Push the tagged image to the Container Registry namespace:
+
+  ```bash
+  sudo docker push <address-of-your-namespace>/mytestapp
+  ```
+
+6. Refresh your Container Registry namespace in the browser to ensure your image has been successfuly pushed.
+
+## Conclusion
+
+You have seen how to containerize an application by creating a Dockerfile, using the Dockerfile to build a Docker image, and then running the containerized application from the image. As well as this, you can then push the image to a container registry, so that it can be pulled by other users who want to run the application. Future tutorials in this series will show you how to create a Kubernetes Kapsule in the Scaleway console, and use that to deploy your containerized application at scale. We'll then go on to cover more advanced topics such as load balancing your application and managing its storage. Don't hesitate to check out some of the links below in the meantime!
```

Comment:
```suggestion
You have seen how to containerize an application by creating a Dockerfile, using the Dockerfile to build a Docker image, and then running the containerized application from the image. As well as this, you can then push the image to a container registry, so that it can be pulled by other users who want to run the application. Future tutorials in this series will show you how to create a Kubernetes Kapsule in the Scaleway Console, and use that to deploy your containerized application at scale. We'll then go on to cover more advanced topics such as load balancing your application and managing its storage. Don't hesitate to check out some of the links below in the meantime!
```